app/data/encrypted_blob_store.go

```diff
@@ -30,3 +30,11 @@ func (bs *EncryptedBlobStore) WriteNX(name string, blob []byte) (bool, error) {
 	}
 	return bs.store.WriteNX(name, encryptedBlob)
 }
+
+func (bs *EncryptedBlobStore) Write(name string, blob []byte) (bool, error) {
+	encryptedBlob, err := compat.Encrypt(blob, bs.encryptionKey)
```

Comment:
It would be easy enough to just use normal AES - partly playing devil sadvocate but I think there is an argument to keep all usage focused on compat.Encrypt for now just to make it easier to switch over all at once when the time comes.  

If we do change here hopefully compat.Encrypt could delegate to the "normal" function for encryption - but that feels worth doing in separate change if we do.  Could maybe do after merge but before releasing a new version of the server.